nav2_bringup/bringup/launch/nav2_tb3_simulation_launch.py

```diff
@@ -140,8 +140,7 @@ def generate_launch_description():
         cmd=['gzclient'],
         cwd=[launch_dir], output='screen')
 
-    urdf = os.path.join(
-        get_package_share_directory('turtlebot3_description'), 'urdf', 'turtlebot3_waffle.urdf')
+    urdf = os.path.join(bringup_dir, 'urdf', 'turtlebot3_waffle.urdf')
```

Comment:
file name ``turtlebot_waffle.urdf`` needs to match with the file name in the launch file.

Reply:
Some PRs man.. you change something, you break 2 more things ;-)